blocks/library/post-title/post-title-edit.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import Editable from '../../editable';
+import PostPermaLink from './post-permalink';
+import { editPost } from '../../../editor/actions';
+
+const PostTitleEdit = ( { title, focus, setFocus, onChange } ) => [
+	focus && <PostPermaLink />,
+	<Editable
+		key="editable"
+		tagName="h1"
+		value={ [ title ] }
+		focus={ focus }
+		onFocus={ setFocus }
+		onChange={ onChange }
+		style={ { textAlign: 'left' } }
+		formattingControls={ [] } /> ];
+
+export default connect( null, dispatch => ( { onChange: title => {
+	dispatch( editPost( { title: title[ 0 ] } ) );
```

Comment:
What is the value of title here? If we were to consider `PostTitleEdit` the "presentational" component, I would think it's `onChange` callback should be called with the value we'd expect to be using, without any further manipulations like we're doing here.